source/sass/wagtail.scss

```diff
@@ -100,3 +100,7 @@
 }
 
 @import "./wagtail/blocks";
+
+.bg-gray {
+  background: $gray-05;
```

Comment:
@mmmavis Is there a better place to put this?

Reply:
Yup! I'm thinking `global.scss`